turbinia/output_manager.py

```diff
@@ -51,24 +51,28 @@ def __init__(self):
     self.is_setup = False
 
   @staticmethod
-  def get_output_writers(task):
+  def get_output_writers(name, uid, remote_only):
     """Get a list of output writers.
 
     Args:
-      task: A TurbiniaTask object
+      name (str): The name of the Request or Job.
```

Comment:
I think we actually want "Task" here instead of "Job" (and below too).

Reply:
Fixed